addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.js

```diff
@@ -13,18 +13,6 @@ const DynamicSnippetProducts = DynamicSnippetCarousel.extend({
     // Private
     //--------------------------------------------------------------------------
 
-    /**
-     * Check product category exist.
-     * @private
-     * @returns {Promise}
-     */
-    _checkCategoryExist: async function (productCategoryId) {
```

Comment:
Hello @nboulif 🙂 ,
We should (may) not remove a function or modify a function signature in stable as custom will break.
But as this as been done recently we can take the bet that no-one has made one. Same for removing async etc.

Reply:
Indeed, thanks for the reminder :wink: